proposals/buildfile.md

```diff
@@ -0,0 +1,186 @@
+# Jib CLI Buildfile Specification
+
+Specification for a YAML buildfile describing building a container image. This buildfile can be
+used by the jib-cli to generate a container. It is translated directly into a buildplan and
+passed to the builder.
+
+```yaml
+apiVersion: jib/v0alpha1
```

Comment:
```suggestion
apiVersion: jib/v1alpha1
```
`alpha` and `beta` are like Maven `-SNAPSHOT`: `v1alphaX` &rarr; `v1betaX` &rarr; `v1`.
Otherwise you're saying when this is complete then it becomes `v0`.

Reply:
oh yeah this was just filler. I'll update.